ppb/sprites.py

```diff
@@ -141,228 +133,127 @@ def rotate(self, degrees):
         self.rotation += degrees
 
 
-class Side(FauxFloat):
-    """
-    Acts like a float, but also has a variety of accessors.
+class RectangleShapeMixin:
     """
-    sides = {
-        LEFT: ('x', -1),
-        RIGHT: ('x', 1),
-        TOP: ('y', 1),
-        BOTTOM: ('y', -1)
-    }
+    A Mixin that provides a rectangular area to sprites.
 
-    def __init__(self, parent: 'SquareShapeMixin', side: str):
-        self.side = side
-        self.parent = parent
+    You should include RectangleShapeMixin before your BaseSprite in your
+    parent classes.
```

Comment:
Why before? Is it mandatory that `RectangleShapeMixin` happen before `BaseSprite`?

Reply:
This was a thing we discussed before? You had said that mixins should come before the base class as a general rule?

Reply:
Yeah, but documenting it this way kinda implies "`RectangleShapeMixin` must be used with `BaseSprite`"? Which is untrue, and if it was true it should inherit from `BaseSprite`?

Reply:
Would "You should include RectangleShapeMixin before you base class in your class definitions"?

Reply:
I don't think it actually matters in this context? Because it doesn't override anything (just assumes certain things are defined somewhere in the MRO), I don't think it matters?